package/android/src/main/java/com/mrousavy/camera/core/CameraSession+Video.kt

```diff
@@ -108,3 +127,17 @@ fun CameraSession.resumeRecording() {
   val recording = recording ?: throw NoRecordingInProgressError()
   recording.resume()
 }
+
+fun CameraSession.startEndOfStream() {
+  setEISMode("2");
+}
+
+fun CameraSession.restoreEndOfStream() {
+  setEISMode(Settings.Global.getString(context.contentResolver, "enable_EIS"));
+}
+
+fun CameraSession.setEISMode(newMode: String) {
+  Log.i(CameraSession.TAG, "EIS: changed from: " + SystemProperties.get("persist.vendor.camera.enableEIS"));
+  SystemProperties.set("persist.vendor.camera.enableEIS", newMode);
```

Comment:
I believe this whole method needs to be wrapped in a try/catch in the case it's running on an earlier version of the firmware (that doesn't have the SELinux rules allowing us to access this system property)